src/components/NcActions/NcActions.vue

```diff
@@ -716,8 +716,8 @@ export default {
 			</NcActions>
 		</p>
 
-		<h2>Tooltip</h2>
-		Has only text and not interactive elements
+		<h2>Toolip</h2>
```

Comment:
Is it on master as well?

Reply:
Did not change this - this is from the merge. So yes

Reply:
Ah noticed the typo :see_no_evil:
That should not be backported